src/main/java/command/CommandType.java

```diff
@@ -0,0 +1,58 @@
+package command;
+
+import java.util.HashMap;
+// 1. command type
+// 1.1 list
+// 1.2 add
+// 1.3 delete
+// 1.3 find
+// 2. action
+// 3. time
+// 3.1 todo
+// 3.2 deadline
+// 3.3 event
+public enum CommandType {
+    LIST,
+    ADD,
+    DONE,
+    DELETE,
+    FIND,
+    BYE,
+    INVALID;
+
+    private static final HashMap<String, CommandType> CommandStr2Type =
```

Comment:
constant names should be separated by underscore or if not it should be in camelCase (in your case both is fine)  and the variable naming could be made more descriptive and improved (change `2` to `to` for a start

Reply:
consider the above comment for your hashmaps in this enum